### ISSUE TITLE
nextcloud integration (how to?)

### ISSUE THREAD

**Issue by whizkaz** [woefe/ShoppingList#51]
I really like the idea and the plain layout of the App.

Nonetheless I can't get the App to run and sync with a nextcloud account.
I read all the related posts but unfortunately couldn't get any useful hints there.
The directory is not on a SD-card. How can I get the app working with nextcloud-sync?

**Comment by parisch** [comment]
Same for me. Just found this app and it seems to be perfect for my needs, but I couldn't get it so sync via nextcloud. Any Ideas?

**Comment by woefe** [comment]
Unfortunatley, synchronisation with nextcloud is not reliable. The Nextcloud app on Android does not work like on the desktop and cannot continuously watch and sync a folder. There is better integration planned (see #17), but currently I don't have much time to work on this project.

The "best" way to sync with Nextcloud is:
1. Create a shoppinglists folder in your Nextcloud
2. Sync this folder to your Nextcloud files app.
3. Go to ShoppingList settings and find this folder on your Android filesystem (e.g. `/storage/emulated/0/Android/media/com.nextcloud.client/nextcloud/user@cloud.kom/shoppinglists`)
4. If you want to sync the lists, open the Nextcloud files app and synchronize the shoppinglist folder. Unfortunately, this is is a manual process. I think in some cases on older versions of Android, marking the folder as favorite can enable automatic synchronisation.

**Comment by woefe** [comment]
I added some info to readme and Github pages site (see 2929249)

**Comment by pled** [comment]
Hi,
I just installed ShoppingList and set it up like this for Nextcloud (not perfect, but better IMO) : 
- I created a ShoppingList folder (1) somewhere on my android for example : Internal storage/ShoppingList
- I created a ShoppingList folder (2) on my Nextcloud server.
- From Android Nextcloud App - Parameters - Auto Upload, I define manually those folders as local (1) and remote (2). I also define the option "What If file already exists" to "Erase existing version" (check text, this is translated from french !)

Doing that way, all .lst files are synced on the server, and I have a copy available in my Nextcloud client after a sync (so in a different folder that the one used by ShoppingList App).
